docs/maps/maps-getting-started.asciidoc

```diff
@@ -164,10 +164,10 @@ View your geospatial data alongside a heat map and pie chart, and then filter th
 When you apply a filter in one panel, it is applied to all panels on the dashboard.
 
 . Click **Add from library** to open a list of panels that you can add to the dashboard.
-. Add **[Logs] Unique Visitor Heatmap** and **[Logs] Bytes distribution** to the dashboard.
+. Add **[Logs] Unique Destination Heatmap** and **[Logs] Bytes distribution** to the dashboard.
 +
 [role="screenshot"]
-image::maps/images/gs_dashboard_with_map.png[]
+image::maps/images/gs_dashboard_with_map.png[Map in a dashboard with 2 other panels]
```

Comment:
This image should be taken before the filter is applied. Right now, its taken after the filter is applied.